boards/avr/at90usb/micropendous3/configs/hello/defconfig

```diff
@@ -13,7 +13,7 @@ CONFIG_ARCH_CHIP="at90usb"
 CONFIG_ARCH_CHIP_AT90USB647=y
 CONFIG_ARCH_CHIP_AT90USB=y
 CONFIG_ARCH_STACKDUMP=y
-CONFIG_AVR_BUILDROOT_TOOLCHAIN=y
+CONFIG_AVR_LINUXGCC_TOOLCHAIN=y
```

Comment:
Should we test buildroot toolchain too?

Reply:
This requires building the NuttX buildroot toolchain which we are not doing for any of the other boards yet either.

Might be worth adding to a nightly build flow or something.